src/Microsoft.Identity.Web.MicrosoftGraph/BaseRequestExtensions.cs

```diff
@@ -15,29 +16,34 @@ public static class BaseRequestExtensions
         /// </summary>
         /// <param name="baseRequest">The <see cref="IBaseRequest"/>.</param>
         /// <param name="scopes">Microsoft graph scopes used to authenticate this request.</param>
-        public static T WithScopes<T>(this T baseRequest, string[] scopes) where T : IBaseRequest
+        public static T WithScopes<T>(this T baseRequest, params string[] scopes) where T : IBaseRequest
         {
-            string authHandlerOptionKey = typeof(TokenAcquisitionAuthenticationProviderOption).ToString();
```

Comment:
@jennyf19 : this was the issue i was strugging with. 
Shouldn't have been TokenAcquisitionAuthenticationProviderOption, but AuthenticationHandlerOption

cc: @darrelmiller FYI

Reply:
i see.

---
In reply to: [509692341](https://github.com/AzureAD/microsoft-identity-web/pull/709#discussion_r509692341) [](ancestors = 509692341)